Doc/library/__future__.rst

```diff
@@ -1,5 +1,5 @@
-:mod:`__future__` --- Future statement definitions
-==================================================
+:mod:`!__future__` --- Future statement definitions
+====================================================
```

Comment:
It looks like in every page, you've added two characters to the "underline" line below the title, but you should have added only one

Reply:
Ugh! Will fix.

Reply:
Fixed.